contracts/examples/BalanceCredentialIssuer.sol

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.16;
+
+import { OwnableUpgradeable } from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { IState } from "@iden3/contracts/interfaces/IState.sol";
+import { ClaimBuilder } from "@iden3/contracts/lib/ClaimBuilder.sol";
+import { IdentityLib } from "@iden3/contracts/lib/IdentityLib.sol";
+import { IdentityBase } from "@iden3/contracts/lib/IdentityBase.sol";
+import { PrimitiveTypeUtils } from "@iden3/contracts/lib/PrimitiveTypeUtils.sol";
+
+/**
+ * @dev Example of decentralized balance credential issuer.
+ * This issuer issue non-merklized credentials decentralized.
+ */
+contract BalanceCredentialIssuer is IdentityBase, OwnableUpgradeable {
+    using IdentityLib for IdentityLib.Data;
+
+    // Information about the schema that is used
+    // for building verifiable credential from claim
+    struct SchemaInfo {
+        string jsonldSchemaURL;
+        string jsonSchemaURL;
+        string credentialType;
+    }
+
+    // Claim representation
+    struct Claim {
+        uint256[8] claim;
+    }
+    
+
+    uint256[500] private __gap_before;
+    
+    // jsonldSchemaHash hash of jsonld schema. 
+    // More about schema: https://devs.polygonid.com/docs/issuer-node/issuer-node-api/claim/apis/#get-claims
+    uint256 private constant jsonldSchemaHash = 134825296953649542485291823871789853562;
```

Comment:
move `constant` before `__gap_before`, and  update gap_aftert to `uint256[48] private __gap_after`,
constant not a part of the storage vars